src/ko/ui/defaultViewManager.html

```diff
@@ -1,15 +1,16 @@
-<workshops></workshops>
-<dropbucket></dropbucket>
+
+<workshops data-bind="visible: !$component.previewMode()"></workshops>
+<dropbucket data-bind="visible: !$component.previewMode()"></dropbucket>
 
 <!-- ko if: host -->
-<div class="viewport" data-bind="host: { viewport: viewport, host: host, block: block }"></div>
+<div class="viewport" data-bind="host: { viewport: viewport, host: host, block: block, previewMode: $component.previewMode }"></div>
 
 <div class="shutter" data-bind="css: { 'active': shutter }">
     <spinner class="fit"></spinner>
 </div>
 <!-- /ko -->
 
-<div class="toasts" data-bind="foreach: toasts">
+<div class="toasts" data-bind="visible: !$component.previewMode(), foreach: toasts">
```

Comment:
I think we can prevent toasts from even triggering in DefaultViewManager. Also, is there something specific that triggers messages in Preview mode you needed to disable?

Reply:
`<div class="toasts" data-bind="visible: !$component.previewMode(), foreach: toasts">`